spec/services/solidus_pay_tomorrow/client/void_service_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SolidusPayTomorrow::Client::VoidService do
+  before do
+    ENV['API_BASE_URI'] = 'https://api-base-url.com'
+  end
+
+  describe '#call' do
+    it 'successful cancellation', vcr: { cassette_name: 'client/void/success' } do
+      response = described_class.call(order_token: 'order_token')
```

Comment:
the service successfully canceled an order with a token `'order_token'`? 🤯

Reply:
No. The test passed with a valid token, but need not have it in version control, hence replaced a valid_token with `order_token` in yml and test.

Is this not the recommended way?

Reply:
Ignore this comment. We decided not to use VCR with this extension